removeS3AccessLogging/function.js

```diff
@@ -1,22 +1,30 @@
 const lambdaCfn = require('@mapbox/lambda-cfn');
+const splitOnComma = require('@mapbox/lambda-cfn').splitOnComma;
 
 module.exports.fn = function(event, context, callback) {
   if (event.detail.errorCode) return callback(null, event.detail.errorMessage);
 
   const requestParameters = event.detail.requestParameters;
   const accessLoggingStatus = requestParameters.BucketLoggingStatus.LoggingEnabled;
 
-  if (accessLoggingStatus === undefined) {
-    const message = {
-      subject: `S3 server access logging removed from ${requestParameters.bucketName}`,
-      summary: `S3 server access logging removed from ${requestParameters.bucketName}`,
-      event: event
-    };
-    lambdaCfn.message(message, (err, result) => {
-      callback(err, result);
-    });
+  const bucketFilter = process.env.bucketFilter ? splitOnComma(process.env.bucketFilter) : [];
+  const filtered = bucketFilter.filter((b) => requestParameters.bucketName.includes(b));
```

Comment:
Is `requestParameters.bucketName` an array that contains more than 1 entry? If not, it would feel more straightforward to 

```js
const ignore = bucketFilter.includes(requestParameters.bucketName);
```

Reply:
Also what about pattern matching? The tests I'm working on now will always be buckets that are named like

```
existing-s3-bucket-test-randomcrap
```

Maybe the parameter entry should be comma-delimited regular expressions? So `existing-s3-bucket-test-.*` and matching criteria like

```js
const ignore = bucketFilter.find((regex) => regex.test(requestParameters.bucketName));
```

Reply:
Yes and yes - thanks for reviewing @rclark!

Reply:
I'd inverted to `requestParameters.bucketName.includes(bucketFilter)` to allow for matching a prefix like `existing-s3-bucket-test-` rather than an entire bucket name, but regex is definitely the more stable solution there.

Reply:
Oh! [string.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) is something that I didn't know existed. TIL!